lib/progress.coffee

```diff
@@ -15,12 +15,17 @@ limitations under the License.
 ###
 
 noop = require('lodash/noop')
-zlib = require('zlib')
-stream = require('stream')
-progress = require('progress-stream')
-rindle = require('rindle')
+once = require('lodash/once')
 utils = require('./utils')
 
+# Lazy-load dependencies to prevent this happening in the browser
+# where this module is not applicable (but still included into the bundle)
+loadDeps = once -> {
```

Comment:
Can you inline these requires into their respective functions, `require` already has the `once` semantics and having it directly in the function also saves the cost to load the modules in cases where the functions requiring them are never used (we gained a decent speed up in the cli by lazy-loading on a per-command basis)